lib/ret/hub.ex

```diff
@@ -59,7 +63,9 @@ defmodule Ret.Hub do
     spawn_drawing: false,
     pin_objects: false,
     spawn_emoji: false,
-    fly: false
+    fly: false,
+    voice_chat: false,
+    text_chat: false
   }
```

Comment:
It seems like there is redundancy between these two module attributes and `@object_actions` in our `Canada.Can` implementations. Not necessary at this moment, but we could set one source of truth for both.

Reply:
I think `@object_actions` enumerates all the things a user might be allowed to do while `@default_member_permissions` and `@default_restrictive_member_permissions` are specific configurations to apply to users. There's a flag in the admin panel called "permissive rooms" that changes the default permissions applied to users across the instance.

https://github.com/mozilla/reticulum/blob/9967e50e3d970e01f8e121fe4f801cf7dae6cd1d/lib/ret/hub.ex#L200-L206

![image](https://user-images.githubusercontent.com/4072106/196983047-4847bda1-77e9-4010-b463-7206a32ea823.png)